docs/serving/spec/knative-api-specification-1.0.md

```diff
@@ -148,6 +148,39 @@ Service object to reduce the cognitive overhead for application developers – t
 Service object should provide sufficient controls to cover most of application
 deployment scenarios (by frequency).
 
+## Extentions
+
+Extending the Knative resource model allows for custom semantics to be
+offered by implementions of the specification. Unless otherwise noted,
+implementations of this specification MAY define extensions but those
+extensions MUST NOT contradict the semantics defined within this specification.
+
+There are several ways in which implementations can extend the model:
+- Annotations and Labels
```

Comment:
I think you want `**` and maybe a `<br>` here for this to render the way you wrote:

https://github.com/knative/docs/blob/f6adfe6791816ed60c35ff6d8c0a3035457d70b4/docs/serving/spec/knative-api-specification-1.0.md#extentions

Reply:
done